src/components/VideoPage/VideoPage.js

```diff
@@ -0,0 +1,124 @@
+import { useState, useContext, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import { RiThumbUpFill, RiPlayListAddFill } from "react-icons/ri";
+import { IoIosShareAlt } from "react-icons/io";
+
+import VideoCard from "../VideoCard/VideoCard";
+import Modal from "../Modal/Modal";
+import PlaylistForm from "../Playlist/PlaylistForm/PlaylistForm";
+import { UserContext } from "../../store/UserContext/UserContext";
+
+import * as userActionTypes from "../../store/types/userActionTypes";
+import { showToast, convertToShortNumber } from "../../utils";
+import { videos } from "../../data";
+
+import "./video-page.css";
+
+export default function VideoPage() {
+  const { user, userDispatch } = useContext(UserContext);
+
+  const { embedId } = useParams();
+
+  const [isPlaylistModalOpen, setIsPlaylistModalOpen] = useState(false);
+
+  const togglePlaylistModal = () => {
+    setIsPlaylistModalOpen(!isPlaylistModalOpen);
+  };
+
+  const copyUrlHandler = () => {
+    navigator.clipboard.writeText(window.location.href);
+    showToast("Link copied to clipboard!");
+  };
+
+  useEffect(() => {
+    userDispatch({
+      type: userActionTypes.UPDATE_HISTORY,
+      payload: {
+        videoEmbedId: embedId,
+      },
+    });
+    window.scroll({
```

Comment:
Would like to know the purpose behind this line?

Reply:
When the user selects a video from the suggestions in the video page, the redirect is done to the same component and hence view stays at the same position of the page. So when there is a change in the video I'm scrolling to the top to provide a good user experience

Reply:
Thanks :)